src/ApiService/ApiService/onefuzzlib/NodeOperations.cs

```diff
@@ -258,6 +355,19 @@ IOnefuzzContext context
         await SendStopIfFree(updatedNode);
     }
 
+    public async Async.Task SetShutdown(Node node) {
+        //don't give out more work to the node, but let it finish existing work
+        _logTracer.Info($"setting delte_requested: {node.MachineId}");
+        node = node with { DeleteRequested = true };
+        var r = await Replace(node);
```

Comment:
prefer `Update` to `Replace`

```suggestion
        var r = await Update(node);
```

Reply:
I see what you mean. 

100% onboard with doing it better than Python implementation (since Python just replaces the record)

Reply:
We need to revisit all this later on, since originally we just did "Replace" to follow the Python implementation.

So there are lots of places where Update should be done instead of Replace